### PR TITLE
Add cloud-provider repo

### PR DESCRIPTION
**Pull request by nikhita**
- Repo creation request: https://github.com/kubernetes/org/issues/147
- Repo with initial empty commit: https://github.com/kubernetes/cloud-provider
- PR against test-infra for branch protection: https://github.com/kubernetes/test-infra/pull/9739
- Dependencies: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/Godeps/Godeps.json

/cc @dims @sttts 
/assign @sttts

### CODE REVIEW

**Comment by nikhita** [comment]
Related: https://github.com/kubernetes/kubernetes/pull/69554

File: configs/kubernetes-rules-configmap.yaml [hunk 1]
```diff
@@ -1023,3 +1023,17 @@ data:
           branch: master
         - repository: api
           branch: master
+    - destination: cluster-bootstrap
```
Comment (sttts) [inline configs/kubernetes-rules-configmap.yaml]: cloud-provider?

Comment (nikhita) [reply configs/kubernetes-rules-configmap.yaml]: :woman_facepalming: had updated with the change locally, but forgot to push

Fixed now.

**Comment by sttts** [comment]
Redeployed.

**Comment by nikhita** [comment]
Publishing was successful :tada: 

https://github.com/kubernetes/cloud-provider

**Comment by dims** [comment]
excellent. thanks!

**Pull request merged**